Tasks/codepush-release-cordova/task.json

```diff
@@ -0,0 +1,173 @@
+            "helpMarkDown": "",
+            "options": {
+                "AccessKey": "Access Key",
+                "ServiceEndpointCodePush": "Service Endpoint (CodePush)",
+                "ServiceEndpointHockeyApp": "Service Endpoint (HockeyApp)"
+            }
+        },
+        {
+            "name": "accessKey",
+            "type": "string",
+            "label": "Access Key",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Access key used to authenticate with the CodePush service.",
+            "visibleRule": "authType = AccessKey"
+        },
+        {
+            "name": "serviceEndpointCodePush",
+            "type": "connectedService:codepush-auth-key",
+            "label": "Service Endpoint",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "CodePush service endpoint that is configured with your account credentials.",
+            "visibleRule": "authType = ServiceEndpointCodePush"
+        },
+        {
+            "name": "serviceEndpointHockeyApp",
+            "type": "connectedService:HockeyApp",
+            "label": "Service Endpoint",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "HockeyApp service endpoint that is configured with your account credentials.",
+            "visibleRule": "authType = ServiceEndpointHockeyApp"
+        },
+        {
+            "name": "appName",
+            "type": "string",
+            "label": "App Name",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Name of the app you want to release the update for."
+        },
+        {
+            "name": "deploymentName",
+            "type": "pickList",
+            "label": "Deployment",
+            "defaultValue": "Staging",
+            "required": false,
+            "helpMarkDown": "Name of the deployment you want to release the update to.",
+            "options": {
+                "Production": "Production",
+                "Staging": "Staging"
+            },
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "platform",
+            "type": "pickList",
+            "label": "Platform",
+            "defaultValue": "android",
+            "required": true,
+            "helpMarkDown": "Platform to build/prepare the release for.",
+            "options": {
+                "android": "Android",
+                "ios": "iOS"
+            }
+        },
+        {
+            "name": "shouldBuild",
+            "type": "boolean",
+            "label": "Build?",
+            "defaultValue": false,
+            "required": true,
+            "helpMarkDown":"This task runs cordova prepare to set up assets by default. Checking this box will tell it to do a cordova build instead of a cordova prepare. Use this if you have a build hook that sets up assets."
```

Comment:
Maybe make this: `By default, this task will trigger "cordova prepare" in order to create your release. This specifies that you'd like to run "cordova build" instead, which can be helpful if your app has build hooks.`